packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## v1.1.0-dev (TTBD)
+- Setting cookies config `samesite` conditionally when the code is loaded in/outside of an iframe [#1403](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1403)
```

Comment:
Can we include mention of the auth loop call? This is the bug that we wanted to fix. 

The line here explains only the implementation of how we fix the bug. But we also need to mention what the bug is too.

Reply:
I wouldn't call the auth loop a bug because it's not how the app intends to run (in a cross origin environment)

Reply:
Oh right, not really a bug yet because the feature is not even out yet. Ok, supporting storefront preview is a better description of the change.